core/arch/instr.h

```diff
@@ -399,6 +414,9 @@ struct _instr_t {
     /* flags contains the constants defined above */
     uint flags;
 
+    /* hints for encoding this instr in a specific way */
```

Comment:
Mention that it holds dr_encoding_hint_type_t

Reply:
Done.